.release.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
remove this

Reply:
removed
![image](https://github.com/deco-sites/start/assets/76822093/a2be334a-8fb3-4a14-b856-14c7ba62a849)